stable/rundeck/values.yaml

```diff
@@ -0,0 +1,128 @@
+# Default values for rundeck.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+job:
+  image:
+    repository: deepakr11/rundeck-job
+    tag: 0.1.0
+    pullPolicy: Always
+
+image:
+  repository: deepakr11/rundeck
+  tag: 0.1.0
+  pullPolicy: Always
+
+service:
+  type: ClusterIP
+  http:
+    port: 4440
+  https:
+    port: 4443
+
+ingress:
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - rundeck.default.svc.cluster.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits:
+    cpu: 1
+    memory: 512Mi
+  requests:
+    cpu: 1
+    memory: 512Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+#####################################################################
+# Application Configurations
+
+#######################################
+# Pod Spec Overrides
+
+appname: rundeck
+
+env:
+  SHARED_STORAGE: /rundeck
+  DB_URL: db.default.svc.cluster.local
+  DB_SCHEMA: rundeck
+  DB_USER: rundeck
+  DB_PASS: rundeck
+  RD_URL: "http://rundeck.default.svc.cluster.local:80/api/18"
+  RD_USER: rundeck
```

Comment:
If we have a finite set of envs we need to pass it would be better to structure them in a cleaner way
```
config:
  database:
     schema:
      user:
```